docs/product-assembly_running_employee_rostering_in_wb/src/main/asciidoc/wb-employee-rostering-balance-employees-shift-rule-proc.adoc

```diff
@@ -0,0 +1,32 @@
+[id='wb-employee-rostering-balance-employees-shift-rule-proc']
+= Create the `BalanceEmployeesShiftNumber` guided rule 
+
+The `BalanceEmployeesShiftNumber` guided rule creates a soft constraint that ensures all employees are assigned a shift, or that all employees are assigned an equal number of shifts. In the employee rostering example, this constraint is created using the guided rule editor. 
+
+image::BalanceEmployeesShiftNumber.png[BalanceEmployeesShiftNumber Guided Rule]
+
+.Procedure
+. Go to *Menu* -> *Design* -> *Projects* and click the project name.
+. Click *Create New Asset* -> *Guided Rule*.
+. Enter `BalanceEmployeesShiftNumber` as the *Guided Rule* name and select the `employeerostering.employeerostering` *Package*. 
+. Click *Ok* to create the rule asset.
+. Add a *WHEN* condition by clicking the image:5686.png[] in the *WHEN* field.
+. Select `Employee` in the *Add a condition to the rule...* window. Click *+Ok*. 
+. Click on the `Employee` condition to modify the constraints and add the variable name `$employee`.
+. Add the *WHEN* condition `From Accumulate...`. 
+.. Above the `From Accumulate...` condition, click *click to add pattern...* and select `Number` as the fact type from the drop down list.
+.. Add the variable name `$shiftCount` to the `Number` condition.
+.. Below the `From Accumulate...` condition, click *click to add pattern...* and select the `ShiftAssignment` fact type from the drop down list.
+.. Add the variable name `$shiftAssignment` to the `ShiftAssignment` fact type.
+.. Click on the `ShiftAssignment` condition again and from the *Add a restriction on a field* drop down list, select `employee`.
+.. Select `equal to` from the drop down list next to the `employee` constraint.
+.. Click the image:6191.png[edit] icon next to the drop down button to add a variable, and click *Bound variable* in the *Field value* window.
+.. Select `$employee` from the drop down list.
+.. In the *Function* box type `count($shiftAssignment)`.
```

Comment:
Technically "type" should be "enter", per IBM SG (for clarity, supposedly). Another super minor thing. Completely your call. If you do change, be sure to update it everywhere else also.